packages/st2/Makefile

```diff
@@ -30,6 +30,10 @@ else
 	DEB_DISTRO := $(shell lsb_release -cs)
 endif
 
+# Makefile function to retry the failed command 'N' times
+# Example: $(call retry,3,some_script.sh)
+retry = $(2) $(foreach t,$(shell seq 1 ${1}),|| (echo -e "\033[33m Failed ($$?): '$(2)'\n Retrying $t ... \033[0m"; $(2)))
```

Comment:
Bash one liner!

I would prefer to make it just a normal function since it's more readable, but whatever :)